style.css

```diff
@@ -235,120 +532,630 @@ body {
     width: 24px;
   }
 
-  .language__arrow {
-    height: 8px;
-    margin: 7px;
-    width: 12px;
+  .language__arrow {
+    height: 8px;
+    margin: 7px;
+    width: 12px;
+  }
+
+  .header-menu {
+    right: 0;
+  }
+
+  .collection {
+    padding-left: 9vw;
+    padding-right: 8vw;
+    padding-top: 50px;
+  }
+
+  .collection__img {
+    margin-right: 4vw;
+    height: 50%;
+    width: 45%;
+  }
+
+  .collection__title {
+    font-size: 61px;
+    line-height: 61px;
+    margin-top: 1.3vw;
+  }
+
+  .collection__description {
+    font-size: 14px;
+    margin-bottom: 1.7vw;
+  }
+
+  .collection__btn {
+    font-size: 12px;
+    margin-right: 1vw;
+    height: 40px;
+    width: 180px;
+  }
+
+  .range__link {
+    font-size: 12px;
+  }
+
+  .order {
+  	padding-bottom: 7vw;
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/LOCO_proj_Boiko/pullRequest?prid=5841754)